source/custom-registry.rst

```diff
@@ -100,11 +100,35 @@ Percona XtraDB Cluster Operator:
 
 
       .. list-table::
-         :widths: 15 30
+         :widths: 15 50
          :header-rows: 1
 
          * - Image
            - Digest
+         * - percona/percona-xtradb-cluster-operator:1.5.0
+           - 99c551f5437ed5f2fe004877c15a4665260969d0923f7c95fbf7a4c9e7305bb1
+         * - percona/percona-xtradb-cluster-operator:1.5.0-haproxy
+           - 6978a9a4a49515fd8be713312221ab72729a220c180663c5bfdd5bd6d3317504
+         * - percona/percona-xtradb-cluster-operator:1.5.0-proxysql
+           - 629f344e6d6b4e2cec95a461cfcba7f48759bb742182c4ebff453707aba77b6b
+         * - percona/percona-xtradb-cluster-operator:1.5.0-pxc8.0-backup
```

Comment:
it should be "percona/percona-xtradb-cluster:8.0.19-10.1-debug"
the same for the latest 5.7